Ports/libopus/patches/0001-Recognize-SerenityOS-as-a-host-platform.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Kenneth Myhra <kennethmyhra@gmail.com>
+Date: Thu, 22 Dec 2022 17:07:29 +0100
+Subject: [PATCH] Recognize SerenityOS as a host platform
+
+---
+ config.sub | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/config.sub b/config.sub
+index 932128b337eeec7cdc60fd8e0f361531e80295f5..1da4fcc025afdbbf675c621f73aebba27f5c27e1 100755
+--- a/config.sub
++++ b/config.sub
+@@ -1417,7 +1417,7 @@ case $os in
+ 	      | -morphos* | -superux* | -rtmk* | -rtmk-nova* | -windiss* \
+ 	      | -powermax* | -dnix* | -nx6 | -nx7 | -sei* | -dragonfly* \
+ 	      | -skyos* | -haiku* | -rdos* | -toppers* | -drops* | -es* \
+-	      | -onefs* | -tirtos* | -phoenix* | -fuchsia* | -redox*)
++	      | -onefs* | -tirtos* | -phoenix* | -fuchsia* | -redox* | -serenity*)
```

Comment:
Just to be sure: `use_fresh_config_sub` did not work here?

Reply:
`use_fresh_config_sub` does indeed work